activedirectory/release_notes.md

```diff
@@ -1,4 +1,3 @@
 #### What's Improved
-- Added the following new operations and playbooks:
-  - Enable Computer
-  - Disable Computer
+- Added the following pluggable enrichment playbooks that automatically enrich data when you install and configure the Active Directory: 
```

Comment:
Please get it reviewed by Abhishek Shukla as well

Reply:
Namrata is working on an enrichment connector doc's and she already reviewed this format for other enrichment connectors.